kitsune/search/views.py

```diff
@@ -490,10 +346,7 @@ def advanced_search(request, template=None):
     offset = (page - 1) * settings.SEARCH_RESULTS_PER_PAGE
 
     lang = language.lower()
-    if settings.LANGUAGES_DICT.get(lang):
-        lang_name = settings.LANGUAGES_DICT[lang]
-    else:
-        lang_name = ''
+    lang_name = settings.LANGUAGES_DICT.get(lang, '') or ''
```

Comment:
I saw this, and then

![http://4.bp.blogspot.com/_D_Z-D2tzi14/TNc4MEFzVRI/AAAAAAAAEBo/1mYo8KmVK_o/s1600/dogs10.png](http://4.bp.blogspot.com/_D_Z-D2tzi14/TNc4MEFzVRI/AAAAAAAAEBo/1mYo8KmVK_o/s1600/dogs10.png)

It makes sense after I realized the `LANGUAGES_DICT` has `None` in it. Sumo has weird stuff in settings. It's probably fine here, especially because you didn't actually change anything. Just weird.